src/plopp/backends/matplotlib/line.py

```diff
@@ -193,12 +205,23 @@ def _make_data(self) -> dict:
             y = sc.concat([y[0:1], y], dim=self._dim)
             if mask is not None:
                 mask['y'] = np.concatenate([mask['y'][0:1], mask['y']])
-        return {
-            'values': {'x': x.values, 'y': y.values},
-            'stddevs': error,
-            'mask': mask,
-            'hist': hist,
-        }
+        return self._set_to_nan_if_not_finite(
```

Comment:
I am wondering if we actually want to do this.
Say for example this was already in plopp. Would you have noticed you had strange values in your data? Probably not. So maybe this is a good thing?

Or maybe it could be activated from a keyword arg? But then that arg would have to be propagated all the way to the top...

Reply:
Yeah that is an important point. I think this feature is too obscure to warrant a keyword argument.

Reply:
After discussion, the conclusion is that we do not make this change, but we can revisit if users start complaining or asking for this fix.

Also note that we need to very very careful about performing any modifications on the data that is given as input. We've had [issues in the past](https://scipp.github.io/reference/developer/adr/0013-plot-should-not-resample-automatically.html) with this, and we concluded that anything that is plotted needs to be an accurate representation of the data that is given to us.